pallets/investments/src/lib.rs

```diff
@@ -459,7 +459,7 @@ pub mod pallet {
 		/// amount is less than the current order, the balance
 		/// will be transferred from the pool to the calling
 		/// account.
-		#[pallet::weight(80_000_000)]
```

Comment:
We definitely need to benchmark this. I looked into whether we needed to do it as part of the current release cycle and determined that the weights here are high enough that the necessarily lookup to fail them is below the hardcoded weight - and they'll always fail, since nobody has permission to invest in pools :sunglasses:

Reply:
https://github.com/centrifuge/centrifuge-chain/issues/918 ^^

I think I used the weight for closing an epoch. Would be surprised if this is above that. So I think this is definetly needed but lower in priority. Wdyt?